cmd/state-tray/internal/libc-compatwrap/compatwrap_linux.go

```diff
@@ -0,0 +1,5 @@
+package compatwrap
```

Comment:
Does this really need to be in its own package? Wouldn't it be sufficient to have the OS suffix?

Reply:
I think so. I was not able to find another way to ensure the C file was only picked up by Linux. I did try adding a `main_linux.go` file with the C import, but the compiler kept trying to add the C file to Windows and Darwin builds regardless and it returned errors "must import C".

Reply:
That's strange.. almost sounds like a bug in Go?